docs/sources/developers/plugins/_index.md

```diff
@@ -48,6 +37,25 @@ Ready to learn more? Check out our other tutorials:
 
 - [Build a panel plugin with D3.js]({{< relref "../../../../../tutorials/build-a-panel-plugin-with-d3.md" >}})
 
+### Guides
+
+Improve an existing plugin with one of our guides:
+
+- [Add support for annotations]({{< relref "add-support-for-annotations.md" >}})
+- [Add support for variables]({{< relref "add-support-for-variables.md" >}})
+- [Build a logs data source]({{< relref "build-a-logs-data-source.md" >}})
+
+### Concepts
+
+Deepen your knowledge through a series of high-level overviews of plugin concepts.
+
+- [Data frames]({{< relref "data-frames.md" >}})
+- [Authentication for data source plugins]({{< relref "authentication.md" >}})
+
+### UI library
+
+Explore the many UI components in our [Grafana UI library](https://developers.grafana.com/ui).
```

Comment:
Do we want to add a link to the Storybook thing as well?

Reply:
Storybook is the framework used to generate the UI library. It's the same thing. I don't think we should refer to it as "Storybook" in our docs, as it's just the technology we use to generate it.